<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwareCheckDialog</class>
 <widget class="QDialog" name="FirmwareCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Check Frequency</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>16</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioEveryLoad">
      <property name="text">
       <string>On each connection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioEveryWeek">
      <property name="text">
       <string>Once per week</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioEveryMonth">
      <property name="text">
       <string>Once per month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioNever">
      <property name="text">
       <string>Never automatically</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>120</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioEveryLoad</tabstop>
  <tabstop>radioEveryWeek</tabstop>
  <tabstop>radioEveryMonth</tabstop>
  <tabstop>radioNever</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FirmwareCheckDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
